coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -2,23 +2,28 @@
 
 
 @generate_repr()
-@generate_eq("documentation", "marker", "range")
+@generate_eq("documentation", "language", "docstyle", "marker", "range")
 class DocumentationComment:
     """
     The DocumentationComment holds information about a documentation comment
     inside source-code, like position etc.
     """
 
-    def __init__(self, documentation, marker, range):
+    def __init__(self, documentation, language,
+                 docstyle, marker=None, range=None):
```

Comment:
I didn't want to make them compulsory.

Like people can do Documentation("/* comment  */, "c", "doxygen") and not worry about other things.